### PR TITLE
add query for sniffer need to execute task list

### PR DESCRIPTION
**Pull request by mrproliu**
add the sniffer need to execute profile task list query

### CODE REVIEW

File: profile/Profile.proto [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+message ProfileTaskCommandQuery {
+    // current sniffer information
+    int32 serviceId = 1;
+    int32 instanceId = 2;
+
+    // current sniffer timestamp
+    int64 time = 3;
+
+    // exists task list
+    repeated string existsTaskList = 4;
```
Comment (wu-sheng) [inline profile/Profile.proto]: Why is this a string only?

File: profile/Profile.proto [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+service ProfileTask {
+
+    // query all sniffer need to execute profile task commands
+    rpc getProfileTaskCommands (ProfileTaskCommandQuery) returns (Commands) {
+    }
+
+}
+
+message ProfileTaskCommandQuery {
```
Comment (wu-sheng) [inline profile/Profile.proto]: Query task should base on agent latest command timestamp, right?

File: profile/Profile.proto [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    // current sniffer information
+    int32 serviceId = 1;
+    int32 instanceId = 2;
+
+    // current sniffer timestamp
+    int64 time = 3;
```
Comment (wu-sheng) [inline profile/Profile.proto]: This is not required. Same as above.

Comment (mrproliu) [reply profile/Profile.proto]: I think service and instance id is required, otherwise query all tasks?

Comment (wu-sheng) [reply profile/Profile.proto]: I mean `time`, the reason same as `Query task should base on agent latest command timestamp`

File: profile/Profile.proto (hunk 2 above)
Comment (mrproliu) [reply profile/Profile.proto]: I think sniffer current timestamp(send the query time) is enough.

**Pull request merged**